bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/HBaseImportOptions.java

```diff
@@ -0,0 +1,23 @@
+package com.google.cloud.bigtable.dataflow;
+
+import com.google.cloud.dataflow.sdk.options.Default;
+import com.google.cloud.dataflow.sdk.options.Description;
+
+/**
+ * An extension of {@link CloudBigtableOptions} that contains additional configuration
+ * for importing HBase sequence files into Cloud Bigtable.
+ */
+public interface HBaseImportOptions extends CloudBigtableOptions {
+  @Description("Prefix or name of the input file(s).")
+  String getFilePattern();
+
+  void setFilePattern(String filePattern);
+
+  @Description(
```

Comment:
Is this going to be more of an enum like set of values for other versions of sequence files?